runtimes/python-3.11/example/requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.27.1
```

Comment:
I don't think this is being installed:

build:

<img width="1178" alt="image" src="https://github.com/open-runtimes/open-runtimes/assets/1477010/3dc78233-6c38-4126-a541-ff7b6553acff">

start:

<img width="1272" alt="image" src="https://github.com/open-runtimes/open-runtimes/assets/1477010/e8b65b55-ebf1-4c29-be8e-6e03b7024ddf">

execute:

<img width="1276" alt="image" src="https://github.com/open-runtimes/open-runtimes/assets/1477010/6ba9548b-4c16-40e8-93c0-2d6bb9548357">

decoded `X-Open-Runtimes-Errors`:

```
Traceback (most recent call last):
  File "/usr/local/server/src/server.py", line 170, in handler
    output = await execute(context)
  File "/usr/local/server/src/server.py", line 150, in execute
    userModule = importlib.import_module("function." + userPath)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/local/server/src/function/main.py", line 2, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

Reply:
Looking at the tests, it looks like we have to pass in a build command that installs the python dependencies. This seems to contradict the readme:

https://github.com/open-runtimes/open-runtimes/blob/1116a08fb3d09b7170e9054a9f68f39388abb8b4/runtimes/python-3.10/README.md?plain=1#L84

Reply:
This should happen during the runtime build phase not the run phase?

Reply:
The build command and build phase I think is automatically handled when function is deployed right @Meldiron ?

Reply:
Updated docs to better explain this.